### PR TITLE
(#826) Update retention policies documentation and images.

### LINKED ISSUE #826: Update documentation for Chocolatey Central Management 0.11.0
### What You Are Seeing?
Documentation needs updating for Chocolatey Central Management 0.11.0.

### What is Expected?
All documentation is updated to reflect Chocolatey Central Management 0.11.0.

### How Did You Get This To Happen? (Steps to Reproduce)
The main documentation that will be effected lives in https://docs.chocolatey.org/en-us/central-management/

### PR DESCRIPTION
**Pull request by corbob**
## Description Of Changes

This updates the images used on the retention settings page to reflect the UI in Chocolatey Central Management 0.11.0. In addition, this adds new documentation about the deployment plan retention feature.

## Motivation and Context

Chocolatey Central Management 0.11.0 release.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [x] Requires a change to menu structure (top or left hand side)/
* [x] Menu structure has been updated

## Related Issue

Fixes #826

### CODE REVIEW

File: input/en-us/central-management/setup/website.md [hunk 1]
```diff
@@ -299,11 +299,11 @@ Here is a copy of items that can be set. They are not required to be encrypted.
 
 ### Step 4.5: Audit Retention
 
-You may want to configure the [automatic deletion of audit logs](xref:ccm-administration-settings-audit-retention) that are captured by Chocolatey Central Management website.
+You may want to configure the [automatic deletion of audit logs](xref:ccm-administration-settings-retention#audit-retention) that are captured by Chocolatey Central Management website.
 
 ### Step 4.6: Stale Computer Deletion
```
Comment (st3phhays) [inline input/en-us/central-management/setup/website.md:304]:
```suggestion
### Step 4.6: Stale Computer Retention

```

We had changed this name to stay in line with the other "Retention"s.

File: input/en-us/central-management/setup/website.md [hunk 2]
```diff
@@ -299,11 +299,11 @@ Here is a copy of items that can be set. They are not required to be encrypted.
 
 ### Step 4.5: Audit Retention
 
-You may want to configure the [automatic deletion of audit logs](xref:ccm-administration-settings-audit-retention) that are captured by Chocolatey Central Management website.
+You may want to configure the [automatic deletion of audit logs](xref:ccm-administration-settings-retention#audit-retention) that are captured by Chocolatey Central Management website.
 
 ### Step 4.6: Stale Computer Deletion
 
-You may want to setup the [automatic deletion of computers](xref:ccm-administration-settings-stale-computer-deletion) that are no longer reporting into Chocolatey Central Management.
+You may want to setup the [automatic deletion of computers](xref:ccm-administration-settings-retention#stale-computer-deletion) that are no longer reporting into Chocolatey Central Management.
```
Comment (st3phhays) [inline input/en-us/central-management/setup/website.md:306]:
```suggestion
You may want to setup the [automatic deletion of computers](xref:ccm-administration-settings-retention#stale-computer-retention) that are no longer reporting into Chocolatey Central Management.

```

File: input/en-us/central-management/usage/website/administration/settings/retention.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Deployment Plan Retention
+
+> :choco-info: **NOTE**
+>
+> This feature is available in Chocolatey Central Management starting with version 0.11.0.
+
+Chocolatey Central Management has the ability to automatically archive or delete deployments that have reached a completion state.
+
+By default, Deployment Plan Retention is disabled for both archiving and deleting. When it is enabled, deployment plans that have completed beyond the set timeframe (the default is 30 days) will be archived, or deleted. If both archive and delete is enabled, the delete period will be considered the combination of the archive and the delete.
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/administration/settings/retention.md:69]:
```suggestion
By default, Deployment Plan Retention is disabled for both archiving and deleting. When it is enabled, Deployment Plans that have completed beyond the set timeframe (the default is 30 days) will be archived, or deleted. If both archive and delete is enabled, the delete period will be considered the combination of the archive and the delete.

```

**Review by st3phhays — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/administration/settings/retention.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+Order: 50
+xref: ccm-administration-settings-retention
+Title: Retention Settings
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/administration/settings/retention.md:4]: Retention Policies

File: input/en-us/central-management/usage/website/administration/settings/retention.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+Chocolatey Central Management has a number of settings that allow you to adjust various retention periods. By default, only the Audit Retention policy is enabled.
+
+If you want to change these settings, follow these steps:
+
+1. Open the CCM Site in the browser.
+1. Login with the `ccmadmin` user.
+1. In the left hand menu click on `Administration` and then `Settings`.
+1. Click on the `Retention` tab
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/administration/settings/retention.md:18]:
```suggestion
1. Click on the `Retention Policies` tab

```

File: input/en-us/central-management/usage/website/administration/settings/retention.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+1. Modify the settings as required
+1. Click the `Save All` button at the top right of the page to save your settings.
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/administration/settings/retention.md:20]:
```suggestion
1. Click the `Save All` button at the bottom left of the page to save your settings.

```

**Review by st3phhays — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/administration/settings/retention.md (hunk 4 above)
Comment (corbob) [reply input/en-us/central-management/usage/website/administration/settings/retention.md:4]:
```suggestion
Title: Retention Policies
```

**Review by st3phhays — APPROVED** [review]
LGTM!

**Pull request merged**